ansible/roles/execut-api/templates/.env.j2

```diff
@@ -0,0 +1,3 @@
+# {{ ansible_managed }}
+DATABASE_URL={{ database_url }}
```

Comment:
I don't think this is a secret, we can just write this out here, right? Or does the application expect the password to be part of the database url?